### DISCUSSION TITLE
Dreame F9 Reset Button

### DISCUSSION THREAD

**Discussion by dominik-b13** [Hypfer/Valetudo#1713] [Q&A / Support] [answered]
I wanted to reset my Dreame F9, because i have a new Wifi etc.. I used the reset Button next to the Wifi LED. Now i cant access the webinterface. Does the Reset Button also reset the firmware ? I didnt find any information about how to reset the Wifi.

**Comment by Hypfer** [comment, marked as answer]
Yep, the reset button does a factory reset that wipes the `/data` partition which contains Valetudo.
To only reset Wi-Fi, press and hold the two buttons at the top until the robot responds with some voice clip about the Wi-Fi being reset.

To recover your rooted but valetudo-less robot, please do the following:

1. download the latest binary for your model of robot: 
    https://github.com/Hypfer/Valetudo/releases/latest/download/valetudo-armv7
3. copy onto the robot so that the binary is /data/valetudo
4. chmod +x /data/valetudo
5. cp /misc/_root_postboot.sh.tpl /data/_root_postboot.sh
6. chmod +x /data/_root_postboot.sh
7. reboot
